sections/map_elements.tex

```diff
@@ -16,7 +16,7 @@
   \transparent{0.2}\includegraphics[width=0.6\linewidth]{\art/earth_magic.png}
 \end{center}
 
-\subsection*{Map Tile Anatomy}\index{Map Tile}
+\subheader{Map Tile Anatomy}\index{Map Tile}
```

Comment:
Considering this is on the same page as Map Elements chapter itself, this seems useless to me